drivers/storage/portworx/portworx_test.go

```diff
@@ -661,6 +661,7 @@ func TestValidatePure(t *testing.T) {
 	<-recorder.Events // Pop first event which is Default telemetry enabled event
 	require.Contains(t, <-recorder.Events,
 		fmt.Sprintf("%v %v %s", v1.EventTypeNormal, util.PassPreFlight, "Enabling PX-StoreV2"))
+	require.Contains(t, *cluster.Spec.CloudStorage.SystemMdDeviceSpec, DefCmetaPure)
```

Comment:
You should verify `-metadata` also

Reply:
Updated the UT to check for -metadata and ran the above job.   You can see the param is passed in correctly 
```
[root@k8sjenkins-agent21 tmp]# ./kubectl exec -it -n kube-system users-naveen-facd-dmthin-px-int-2clq9 -- /opt/pwx/bin/px-runc info 
Defaulted container "portworx" out of: portworx, csi-node-driver-registrar
INFO[0000] Detected numCPUs: 0                          
PX-RunC information
-------------------
           Version : 3.1.0-ea-b6014ef
    Install params : /opt/pwx/bin/px-runc install -c users-naveen-facd-dmthin-px-int -x kubernetes -b -cloud_provider pure -s size=256 -metadata size=64 -secret_type k8s --tracefile-diskusage=5 -memory 4294967296 -T px-storev2 -v /var/lib/kubelet:/var/lib/kubelet:shared -v /opt/pwx/oci/mounts/etc/ccm:/etc/ccm -v /opt/pwx/oci/mounts/etc/hosts:/etc/hosts -v /opt/pwx/oci/mounts/etc/resolv.conf:/etc/resolv.conf -v /opt/pwx/oci/mounts/tmp/px-termination-log:/tmp/px-termination-log -v /var/cores:/var/cores -v /var/lib/containerd:/var/lib/containerd -v /var/lib/osd:/var/lib/osd:rshared -v /var/run/dbus:/var/run/dbus -v /opt/pwx/oci/mounts/var/run/secrets/kubernetes.io/serviceaccount:/var/run/secrets/kubernetes.io/serviceaccount -e COREDNS_PORT=udp://10.233.0.3:53 -e COREDNS_PORT_53_TCP=tcp://10.233.0.3:53 -e COREDNS_PORT_53_TCP_ADDR=10.233.0.3 -e COREDNS_PORT_53_TCP_PORT=53 -e COREDNS_PORT_53_TCP_PROTO=tcp -e COREDNS_PORT_53_UDP=udp://10.233.0.3:53 -e COREDNS_PORT_53_UDP_ADDR=10.233.0.3 -e COREDNS_PORT_53_UDP_PORT=53 -e COREDNS_PORT_53_UDP_PROTO=udp -e COREDNS_PORT_9153_TCP=tcp://10.233.0.3:9153 -e COREDNS_PORT_9153_TCP_ADDR=10.233.0.3 -e COREDNS_PORT_9153_TCP_PORT=9153 -e COREDNS_PORT_9153_TCP_PROTO=tcp -e COREDNS_SERVICE_HOST=10.233.0.3 -e COREDNS_SERVICE_PORT=53 -e COREDNS_SERVICE_PORT_DNS=53 -e COREDNS_SERVICE_PORT_DNS_TCP=53 -e COREDNS_SERVICE_PORT_METRICS=9153 -e CSI_ENDPOINT=unix:///var/lib/kubelet/plugins/pxd.portworx.com/csi.sock -e DEFAULT_OSB_URL=https://master-staging.portworx.dev/osb/billing/v1/register -e KUBERNETES_PORT=tcp://10.233.0.1:443 -e KUBERNETES_PORT_443_TCP=tcp://10.233.0.1:443 -e KUBERNETES_PORT_443_TCP_ADDR=10.233.0.1 -e KUBERNETES_PORT_443_TCP_PORT=443 -e KUBERNETES_PORT_443_TCP_PROTO=tcp -e KUBERNETES_SERVICE_HOST=10.233.0.1 -e KUBERNETES_SERVICE_PORT=443 -e KUBERNETES_SERVICE_PORT_HTTPS=443 -e NODE_NAME=users-naveen-facd-dmthin-6-1 -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin -e PORTWORX_API_PORT=tcp://10.233.63.184:9001 -e PORTWORX_API_PORT_9001_TCP=tcp://10.233.63.184:9001 -e PORTWORX_API_PORT_9001_TCP_ADDR=10.233.63.184 -e PORTWORX_API_PORT_9001_TCP_PORT=9001 -e PORTWORX_API_PORT_9001_TCP_PROTO=tcp -e PORTWORX_API_PORT_9020_TCP=tcp://10.233.63.184:9020 -e PORTWORX_API_PORT_9020_TCP_ADDR=10.233.63.184 -e PORTWORX_API_PORT_9020_TCP_PORT=9020 -e PORTWORX_API_PORT_9020_TCP_PROTO=tcp -e PORTWORX_API_PORT_9021_TCP=tcp://10.233.63.184:9021 -e PORTWORX_API_PORT_9021_TCP_ADDR=10.233.63.184 -e PORTWORX_API_PORT_9021_TCP_PORT=9021 -e PORTWORX_API_PORT_9021_TCP_PROTO=tcp -e PORTWORX_API_PORT_9023_TCP=tcp://10.233.63.184:9023 -e PORTWORX_API_PORT_9023_TCP_ADDR=10.233.63.184 -e PORTWORX_API_PORT_9023_TCP_PORT=9023 -e PORTWORX_API_PORT_9023_TCP_PROTO=tcp -e PORTWORX_API_SERVICE_HOST=10.233.63.184 -e PORTWORX_API_SERVICE_PORT=9001 -e PORTWORX_API_SERVICE_PORT_PX_API=9001 -e PORTWORX_API_SERVICE_PORT_PX_API_TLS=9023 -e PORTWORX_API_SERVICE_PORT_PX_REST_GATEWAY=9021 -e PORTWORX_API_SERVICE_PORT_PX_SDK=9020 -e PORTWORX_DISABLE_CSI_ALPHA=true -e PORTWORX_KVDB_SERVICE_PORT=tcp://10.233.24.201:9019 -e PORTWORX_KVDB_SERVICE_PORT_9019_TCP=tcp://10.233.24.201:9019 -e PORTWORX_KVDB_SERVICE_PORT_9019_TCP_ADDR=10.233.24.201 -e PORTWORX_KVDB_SERVICE_PORT_9019_TCP_PORT=9019 -e PORTWORX_KVDB_SERVICE_PORT_9019_TCP_PROTO=tcp -e PORTWORX_KVDB_SERVICE_SERVICE_HOST=10.233.24.201 -e PORTWORX_KVDB_SERVICE_SERVICE_PORT=9019 -e PORTWORX_KVDB_SERVICE_SERVICE_PORT_PX_KVDB=9019 -e PORTWORX_OPERATOR_METRICS_PORT=tcp://10.233.34.83:8999 -e PORTWORX_OPERATOR_METRICS_PORT_8999_TCP=tcp://10.233.34.83:8999 -e PORTWORX_OPERATOR_METRICS_PORT_8999_TCP_ADDR=10.233.34.83 -e PORTWORX_OPERATOR_METRICS_PORT_8999_TCP_PORT=8999 -e PORTWORX_OPERATOR_METRICS_PORT_8999_TCP_PROTO=tcp -e PORTWORX_OPERATOR_METRICS_SERVICE_HOST=10.233.34.83 -e PORTWORX_OPERATOR_METRICS_SERVICE_PORT=8999 -e PORTWORX_OPERATOR_METRICS_SERVICE_PORT_METRICS=8999 -e PORTWORX_SERVICE_PORT=tcp://10.233.48.250:9001 -e PORTWORX_SERVICE_PORT_9001_TCP=tcp://10.233.48.250:9001 -e PORTWORX_SERVICE_PORT_9001_TCP_ADDR=10.233.48.250 -e PORTWORX_SERVICE_PORT_9001_TCP_PORT=9001 -e PORTWORX_SERVICE_PORT_9001_TCP_PROTO=tcp -e PORTWORX_SERVICE_PORT_9020_TCP=tcp://10.233.48.250:9020 -e PORTWORX_SERVICE_PORT_9020_TCP_ADDR=10.233.48.250 -e PORTWORX_SERVICE_PORT_9020_TCP_PORT=9020 -e PORTWORX_SERVICE_PORT_9020_TCP_PROTO=tcp -e PORTWORX_SERVICE_PORT_9021_TCP=tcp://10.233.48.250:9021 -e PORTWORX_SERVICE_PORT_9021_TCP_ADDR=10.233.48.250 -e PORTWORX_SERVICE_PORT_9021_TCP_PORT=9021 -e PORTWORX_SERVICE_PORT_9021_TCP_PROTO=tcp -e PORTWORX_SERVICE_PORT_9023_TCP=tcp://10.233.48.250:9023 -e PORTWORX_SERVICE_PORT_9023_TCP_ADDR=10.233.48.250 -e PORTWORX_SERVICE_PORT_9023_TCP_PORT=9023 -e PORTWORX_SERVICE_PORT_9023_TCP_PROTO=tcp -e PORTWORX_SERVICE_SERVICE_HOST=10.233.48.250 -e PORTWORX_SERVICE_SERVICE_PORT=9001 -e PORTWORX_SERVICE_SERVICE_PORT_PX_API=9001 -e PORTWORX_SERVICE_SERVICE_PORT_PX_API_TLS=9023 -e PORTWORX_SERVICE_SERVICE_PORT_PX_REST_GATEWAY=9021 -e PORTWORX_SERVICE_SERVICE_PORT_PX_SDK=9020 -e PURE_FLASHARRAY_SAN_TYPE=ISCSI -e PX_CH_DEFAULT=true -e PX_CH_URL=http://10.13.29.149:8080 -e PX_ENABLE_CACHE_FLUSH=true -e PX_IMAGE=portworx/px-base-enterprise:3.1.0-ea-b6014ef-dev -e PX_LOGLEVEL=debug -e PX_NAMESPACE=kube-system -e PX_RELEASE_MANIFEST_URL=https://edge-install.portworx.com/3.1.0/version -e PX_SECRETS_NAMESPACE=kube-system -e PX_TEMPLATE_VERSION=v4 -e REGISTRY_PASS=dckr_pat_kFcP7wAdNePCURJbpNhcf4QIoLQ -e REGISTRY_SECRET=regcred -e REGISTRY_USER=cnbuautomation800 -e STORK_SERVICE_PORT=tcp://10.233.56.46:8099 -e STORK_SERVICE_PORT_443_TCP=tcp://10.233.56.46:443 -e STORK_SERVICE_PORT_443_TCP_ADDR=10.233.56.46 -e STORK_SERVICE_PORT_443_TCP_PORT=443 -e STORK_SERVICE_PORT_443_TCP_PROTO=tcp -e STORK_SERVICE_PORT_8099_TCP=tcp://10.233.56.46:8099 -e STORK_SERVICE_PORT_8099_TCP_ADDR=10.233.56.46 -e STORK_SERVICE_PORT_8099_TCP_PORT=8099 -e STORK_SERVICE_PORT_8099_TCP_PROTO=tcp -e STORK_SERVICE_SERVICE_HOST=10.233.56.46 -e STORK_SERVICE_SERVICE_PORT=8099 -e STORK_SERVICE_SERVICE_PORT_EXTENDER=8099 -e STORK_SERVICE_SERVICE_PORT_WEBHOOK=443 -e container=oci -e CONTAINER_RUNTIME=docker -e PX_IMAGE_DIGEST=sha256:57368f6ff900de1d939fa159bab46fd3946cc6ba158eb1076b8604b5532beb40 -e KUBELET_DIR=/var/lib/kubelet
     OCI directory : /opt/pwx/oci
    Service status : active
     Container PID : 5346
       Running for : 4m43.1s
```